packages/pwa-storefront-mcp/src/utils/utils.js

```diff
@@ -9,8 +9,10 @@ import path from 'path'
 import {spawn} from 'cross-spawn'
 import {zodToJsonSchema} from 'zod-to-json-schema'
 import {z} from 'zod'
-import os from 'os'
-import {exec} from 'child_process'
+
+// CONSTANTS
+// const CREATE_APP_VERSION = 'latest'
+const CREATE_APP_VERSION = '3.11.0-nightly-20250710080214'
```

Comment:
Once pwa-kit released a new version we can update this, but it's not a big deal to leave that here now. 

We can also thing about making the create-app a direct dependency, but I think there was a reason we didn't go down that path before.

Reply:
Yes we can remove this and replace it with the line above in anticipation for the release of 3.11 by simply assigning `latest`. But the release was pushed again, to tuesday jul 15. 

The plan was to do this in the "productize" ticket.